lib/cybersource/cyber_source_on_demand.rb

```diff
@@ -94,6 +94,10 @@ def empty?
           @response.params['merchantReferenceCode'].nil?
         end
 
+        def request_id
+          @response.params['requestID']
```

Comment:
The request id should be the last transaction id that occurred from CYBS' POV (see [here](https://github.com/killbill/killbill-cybersource-plugin/blob/e9a3d8a43dd15bb9d4fb180e3a4c010934e38fc0/lib/cybersource/cyber_source_on_demand.rb#L156))